client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/index.tsx

```diff
@@ -1,197 +0,0 @@
-		} );
-		if ( ! shouldPreventTicketCreation ) {
-			sendTicket( {
-				locale,
-				from_url: fromUrl,
-				blog_url: siteSlug,
-			} );
-		}
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [ shouldPreventTicketCreation, config, fromUrl, siteSlug ] );
-	let whatToExpect: WhatToExpectProps[] = [];
-
-	if ( shouldPreventTicketCreation ) {
-		whatToExpect = [
-			{
-				icon: envelope,
-				text: __(
-					"We'll send you an email with more details on the process and we'll let you know when we start the migration."
-				),
-			},
-			{
-				icon: group,
-				text: __(
-					"We'll bring over a copy of your site, without affecting the current live version."
-				),
-			},
-			{
-				icon: scheduled,
-				text: __(
-					"We'll send you an update within 2–3 business days. You can also check the progress of your migration from your Sites dashboard."
-				),
-			},
-		];
-	} else {
-		whatToExpect = [
-			{
-				icon: shield,
-				text: __( "We'll explain how to securely share your site credentials with us." ),
-			},
-			{
-				icon: backup,
-				text: __(
-					"We'll update you on the progress of the migration, which usually takes 2–3 business days."
-				),
-			},
-			{
-				icon: group,
-				text: __( "We'll create a copy of your live site, allowing you to compare the two." ),
-			},
-		];
-	}
-
-	whatToExpect.push( {
-		icon: globe,
-		text: __( "We'll help you switch your domain over after the migration is complete." ),
-	} );
-
-	const title = __( "We've received your migration request" );
-
-	const sitesDashboardButton = (
-		<div className="migration-message__cta-wrapper">
-			<Button
-				className="migration-message__cta"
-				href={ '/overview/' + siteSlug }
-				variant="primary"
-				onClick={ () =>
-					recordTracksEvent( 'calypso_migration_message_view_sites_dashboard_click' )
-				}
-			>
-				{ __( 'View Sites dashboard' ) }
-			</Button>
-		</div>
-	);
-
-	const subHeaderText = isPending ? (
-		<LoadingEllipsis />
-	) : (
-		<>
-			{ createInterpolateElement(
-				sprintf(
-					// translators: %(email)s is the customer's email and %(webSite)s his site.
-					__(
-						'You are all set! Our Happiness Engineers will be reaching out to you shortly at <strong>%(email)s</strong> to help you migrate <strong>%(webSite)s</strong> to WordPress.com.'
-					),
-					{
-						email: user?.email,
-						// Strip protocol and trailing slash.
-						webSite: urlToDomainAndPath( fromUrl ),
-					}
-				),
-				{
-					strong: createElement( 'strong' ),
-				}
-			) }
-		</>
-	);
-
-	return (
-		<StepContainer
-			stepName="migration-message"
-			hideBack
-			formattedHeader={
-				<FormattedHeader align="center" headerText={ title } subHeaderText={ subHeaderText } />
-			}
-			isHorizontalLayout={ false }
-			stepContent={
-				<>
-					<h3>{ __( 'What to expect' ) }</h3>
-					{ whatToExpect.map( ( { icon, text }, index ) => (
-						<div key={ index } className="feature">
```

Comment:
Glad we removed those generic classnames 😅 

cc @WBerredo I see we have a few instances in the performance profiler:

https://github.com/Automattic/wp-calypso/blob/dcc83970edf7d7b17118d3b1ffff33bd5a644f12/client/performance-profiler/components/migration-banner/index.tsx

Care to clean them up / prefix them so they're not so generic/common?